pkg/storkctl/applicationbackup.go

```diff
@@ -65,7 +65,7 @@ func newCreateApplicationBackupCommand(cmdFactory Factory, ioStreams genericclio
 			}
 
 			if len(resourceTypes) > 0 {
-				resourceList, err := getResourceTypes(resourceTypes, ioStreams)
+				resourceList, err := getResourceTypes(resourceTypes, ioStreams, cmdFactory)
```

Comment:
nit: can you keep iostreams as last param just to be in sync with other functions?

Reply:
Will address